.eslintrc.js

```diff
@@ -6,18 +6,16 @@ module.exports = {
     jquery: true,
   },
 
-  extends: [
```

Comment:
False diff in pull request due to formatting, this can be a problem if you don't apply standard formatting when multiple people work on same code. Personal formatting is [still possible](https://stackoverflow.com/a/59381335/1641941)